wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,57 +1,83 @@
-| ![](/wiki/shared/judgement/osu!/hit50.png "50") | 50 | The third best achievable result for a hit object, the last one before a miss. Usually denoted with an orange colour. Scoring a 50 will prevent the appearance of a Katu or a Geki at the combo's end. |
-| ![](/wiki/shared/judgement/osu!/hit0.png "Miss") | Miss | The worst possible result for a hit object. Scoring a Miss will reset the current combo to 0 and will prevent the appearance of a Katu or a Geki at the combo's end. |
+## Judgements
+
+A **judgement** (a.k.a. **hit result**) is the outcome of interacting with a [hit object](/wiki/Gameplay/Hit_object) during its hit window. Score and accuracy are calculated based on which judgements are received.
+
+These are commonly referred to by their score value (except for misses), i.e. a Great is usually called a "300" and so on.
+
+| Image | Name | [Hit value](/wiki/Gameplay/Score/ScoreV1/osu!) | [Accuracy](/wiki/Gameplay/Accuracy#osu!) | Hit window (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!/hit300.png) | Great | 300 | 100% | `80 - 6 * OD` |
+| ![](/wiki/shared/judgement/osu!/hit100.png) | Ok | 100 | 33.33% | `140 - 8 * OD` |
+| ![](/wiki/shared/judgement/osu!/hit50.png) | Meh | 50 | 16.67% | `200 - 10 * OD` |
+| ![](/wiki/shared/judgement/osu!/hit0.png) | Miss | 0 | 0% | `400` |
+
+The hit window depends on the beatmap's [overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty). A hit is then considered inside a hit window if `hit error < hit window`, meaning the value listed is half of the hit window width.
+
+The hit error and hit window are rounded to the nearest integer, meaning the window is effectively 0.5 ms shorter on both sides than what the formulas suggest.
+
+### Geki and Katu
+
+The last hit object in a [combo set](/wiki/Beatmapping/Combo) may give a variant of the regular judgements depending on which ones were achieved in the combo. They give a bit more [health](/wiki/Gameplay/Health) than the regular ones.
+
+| Image | Name | Variant of | Requirement |
+| :-: | :-: | :-: | :-- |
+| ![](/wiki/shared/judgement/osu!/hit300g.png) | Geki (激) | Great | A Great on all hit objects in a combo |
+| ![](/wiki/shared/judgement/osu!/hit300k.png) | Katu or Katsu (喝) | Great | At least an Ok on all hit objects in a combo and a Great for the last one |
+| ![](/wiki/shared/judgement/osu!/hit100k.png) | Katu or Katsu (喝) | Ok | At least an Ok on all hit objects in a combo |
+
+This mechanic is in reference to [Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan) and [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents), the latter using the terms Elite Beat! and Beat! instead of Geki and Katsu.
```

Comment:
no matter how useless it is, it could have a link to https://osu.ppy.sh/wiki/en/iNiS_games in one form or another

Reply:
that article is so half-finished it feels less productive to link to it >.<

i'd suggest deleting it because any spot that mentions the games kinda already explains the gist of "osu is inspired by these iNiS games" perfectly, and idk what more could even be added to that article other than copying wikipedia